Editor/Utils/TextureUtils.cs

```diff
@@ -66,6 +69,14 @@ static bool IsSolidColor(Texture2D texture)
                 return false;
             }
 
+            var pixel1 = texture.GetPixel(0, 0);
+            var pixel2 = texture.GetPixel(1, 0);
```

Comment:
This seems a good way to early out, however, what happens if the texture width is 1?
please also add a comment.